src/components/Loader/index.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const Loader = ({ color }) => (
+  <div className='loader'>
+    <svg
+      width='120'
+      height='30'
+      viewBox='0 0 120 30'
+      xmlns='http://www.w3.org/2000/svg'
+      fill={color}
```

Comment:
I wonder if we might improve this later (not for this PR) by removing this forced color and using `'currentColor'`